test/ParamValue_test.cpp

```diff
@@ -324,16 +324,16 @@ TEST(ParamValue, conversion_vector_int_operator) {
   std::vector<int> il = {1, 2};
   tdl::ParamValue d(il);
   std::vector<int> il_op = d;
-  EXPECT_TRUE(il_op == il);
-  //TEST_EXCEPTION(Exception::ConversionError, std::vector<string> sl = ParamValue("abc,ab");)
+  EXPECT_EQ(il_op == il, true);
+  EXPECT_THROW(std::vector<std::string> sl = tdl::ParamValue("abc, ab"), tdl::ConversionError);
 }
 
 TEST(ParamValue, toIntVector) {
   std::vector<int> il = {1, 2};
   tdl::ParamValue d(il);
   std::vector<int> il_op = d.toIntVector();
-  EXPECT_TRUE(il_op == il);
-  //TEST_EXCEPTION(Exception::ConversionError, std::vector<string> sl = ParamValue("abc,ab").toStringVector();)
+  EXPECT_EQ(il_op == il, true);
```

Comment:
looks like a rebasing artifact from the test refactoring?

Reply:
I don't know why I did that.... I removed it :-)